load_Data.py

```diff
@@ -0,0 +1,35 @@
+import glob
+import os
+
+import pandas as pd
+
+def get_columns(df, excluding):
+    columns = df.columns
+    return list(set(columns) - set(excluding))
+
+def batch_import_csv(paths):
+    f = "python /Users/ahereza/Work/CodeForAfrica/Takwimu/manage.py importcsv {} --table {} --geo_version 2009 --columns {}"
```

Comment:
Not sure about this file and these paths specifically

Reply:
> Looks good @phillipahereza.
> 
> * Some vizs have very few data points so may be they should only be half the size?

@kilemensi but some other countries have more data points that's why I made the size bigger so as to gracefully handle those situations